agent/tsunami_agent.py

```diff
@@ -70,7 +70,7 @@ def __init__(self,
     def _check_asset_was_added(self, targets) -> bool:
         """Check if the asset was scanned before or not"""
         if targets.domain is not None:
-            if self.set_add(b'agent_wappalyzer_asset', f'{targets.domain}'):
+            if self.set_add(b'agent_tsunami', f'{targets.domain}'):
```

Comment:
should be consistent with the one below: str or bytes.. I think all should be bytes.